todocli/task.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from enum import Enum
+
+from todocli.todo_api_util import api_timestamp_to_datetime
+
+
+class Task:
+    class Status(Enum):
+        Completed = "completed"
+        NotStarted = "notStarted"
+        InProgress = "inProgress"
+        WaitingOnOthers = "waitingOnOthers"
+        Deferred = "deferred"
+
+    class Importance(Enum):
+        Low = "low"
+        Normal = "normal"
+        High = "high"
+
+    def __init__(self, query_result_task):
+        self.title: str = query_result_task["title"]
+        self.id: str = query_result_task["id"]
+        self.importance: Task.Importance = Task.Importance(
+            query_result_task["importance"]
+        )
+        self.status: Task.Status = Task.Status(query_result_task["status"])
```

Comment:
This would also change to 
```suggestion
        self.status = Task.Status[query_result_task["status"]]
```

Reply:
I removed explicitly specifying the type